google/cloud/aiplatform/jobs.py

```diff
@@ -551,6 +558,23 @@ def create(
                 but too high value will result in a whole batch not fitting in a machine's memory,
                 and the whole operation will fail.
                 The default value is 64.
+            model_monitoring_objective_config (aiplatform.model_monitoring.ObjectiveConfig):
+                Optional. The objective config for model monitoring. Passing this parameter enables
+                monitoring on the model associated with this batch prediction job.
+            model_monitoring_alert_config (aiplatform.model_monitoring.EmailAlertConfig):
+                Optional. Configures how model monitoring alerts are sent to the user. Right now
```

Comment:
Are these docstrings copied from the source at https://github.com/googleapis/googleapis/tree/master/google/cloud/aiplatform?  Will we be able to remember to update this when/if alerts other than email alert become supported?

Reply:
They're not directly copied from GAPIC. But Jing's team also confirmed that there's no plans for additional alert configs.